src/gcp_scanner/scanner.py

```diff
@@ -269,7 +270,8 @@ def crawl_loop(initial_sa_tuples: List[Tuple[str, Credentials, List[str]]],
 
       sa_results_data = json.dumps(sa_results, indent=2, sort_keys=False)
 
-      with open(out_dir + '/%s.json' % project_id, 'a',
+      scan_time_suffix = datetime.now().strftime('%Y-%m-%d_%H-%M-%S')
+      with open(f'{out_dir}/{project_id}-{scan_time_suffix}.json', 'a',
```

Comment:
I guess adding another variable for the filename would add to the readability of the program.

```suggestion
      # generate time to append to output filename
      scan_time_suffix = datetime.now().strftime('%Y-%m-%d_%H-%M-%S')
      # generate the filename and append the time suffix (to make the filename unique for every scan) 
      output_filename = f'{out_dir}/{project_id}-{scan_time_suffix}' 
      with open(f'{output_filename}.json', 'a',
```

Separating this in multiple lines is just a personal preference, but I definitely think that it adds to the readability of the script.

Reply:
@peb-peb agreed, added the suggestion in latest commit :)